recipes/erikzenker-hsm/all/conanfile.py

```diff
@@ -1,51 +1,71 @@
-from conan import ConanFile, tools
-from conan.tools.cmake import CMake, cmake_layout
+from conan import ConanFile
 from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
 from conan.tools.scm import Version
 import os
 
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.47.0"
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"
```
The `from conan.tools.build import check_min_cppstd` is only available since 1.50.0. Any older version will raise ImportError.